### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Tagadda**
## Problem

- v2 #98 
- Upgrade to v2023.4.4 #101

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Tagadda** [comment]
!testme

**Comment by Tagadda** [comment]
!testme

**Comment by Tagadda** [comment]
Hold the merge!
Looking at [this comment](https://github.com/YunoHost-Apps/dokuwiki_ynh/pull/103#issuecomment-1739524891), this is a downgrade ?

File: manifest.toml [hunk 1]
```diff
@@ -55,8 +55,8 @@ ram.runtime = "50M"
 
 [resources]
         [resources.sources.main]
-        url = "https://github.com/dokuwiki/dokuwiki/releases/download/release-2023-04-04a/dokuwiki-2023-04-04a.tgz"
-        sha256 = "153c99cf42b9068b1ec21a2c765b862a44b374ad2f1a39223f5511a982b160bb"
+        url = "https://github.com/dokuwiki/dokuwiki/archive/refs/tags/release-2023-04-04.tar.gz"
```
Comment (Gofannon) [inline manifest.toml:58]: Please revert this. release must be `2023-04-04a` (until a new security release)

I don't know which "download link" should be used

- https://github.com/dokuwiki/dokuwiki/releases/download/release-2023-04-04a/dokuwiki-2023-04-04a.tgz
  - from https://github.com/dokuwiki/dokuwiki/releases
- https://download.dokuwiki.org/src/dokuwiki/dokuwiki-2023-04-04a.tgz
  - from https://download.dokuwiki.org/archive

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**